src/providers/ark-api/ark-api.ts

```diff
@@ -126,10 +127,13 @@ export class ArkApiProvider {
 
     // Fallback if the fetchEpoch fail
     this._network.epoch = arktsConfig.blockchain.date;
+    // Fallback if the fetchActiveDelegates fail
+    this._network.activeDelegates = 51;
     this.userDataProvider.onUpdateNetwork$.next(this._network);
 
     this.fetchFees().subscribe();
     this.fetchEpoch().subscribe();
+    this.fetchActiveDelegates().subscribe();
```

Comment:
Don't need to implement a new method. The node configuration has already been fetched to set the vendorField length in the `updateNetwork` method:

https://github.com/ArkEcosystem/mobile-wallet/blob/develop/src/providers/ark-api/ark-api.ts#L471-L476

Reply:
Hey, I think the new commit should resolve this. Not sure how to link the commit here.